### DISCUSSION TITLE
Official Tags for Directory

### DISCUSSION THREAD

**Discussion by kquirapas** [web3phl/directory#30] [Brainstorm]
Let's discus the official final list of tags to be used for the directory

**Comment by warengonzaga** [comment]
We need to make a glossary for this I guess, like adding a new page in the Directory to define these.

w2e = workout to earn
p2e = play to earn
m2e = move to earn
pae = play and earn
f2p = free to play
nft = project that involves nft
staking - project that provides staking
metaverse - project that provides metaverse feature
technology = project that provides technology innovation
solution = a web 3 solution project
service = a web 3 service project
community = community engagement project
dao = dao project
guild = guild project
defi = tackles decentralized finance
wallet = web3 wallet
collection = collection project
gaming = gaming project
arts = focus on arts
streaming = streaming platform on web3
news = news content delivery
media = media content delivery

---

This is from Allan. 👀 Any thoughts?

**Reply by kquirapas** [reply]
@itsnalla

**Reply by jnebab** [reply]
we can add a new tab  or a large modal to show these. also, can add tooltip text pag hinover yung tag.

**Reply by kquirapas** [reply]
G with tooltip

**Comment by jnebab** [comment]
can we also limit the tags that can be added during signup? just to avoid overpopulating the card and avoid overlaps with the card container. @warengonzaga @kquirapas

**Reply by warengonzaga** [reply]
We need to inform @itsnalla here as well to limit.

**Reply by chefallan** [reply]
I suggest limiting the tags to 3 only.

**Reply by jnebab** [reply]
we can go at least 3 and max of 5 tags.

**Reply by chefallan** [reply]
> we can go at least 3 and max of 5 tags.

LFTM

**Reply by warengonzaga** [reply]
I agree, I'll create an issue for this.

**Comment by chefallan** [comment]
w2e = workout to earn
p2e = play to earn
m2e = move to earn
pae = play and earn
f2p = free to play
nft = project that involves nft
staking - project that provides staking
metaverse - project that provides metaverse feature
technology = project that provides technology innovation
solution = a web 3 solution project
service = a web 3 service project
community = community engagement project
dao = dao project
guild = guild project
defi = tackles decentralized finance
wallet = web3 wallet
collectibles= nft collection project
gaming = gaming project
arts = focus on arts
streaming = streaming platform on web3
news = news content delivery
multimedia = multimedia company
marketplace = marketplace for nft
launchpad = launchpad for projects
organization = organization
vc = venture capital company
ai - ai oriented project
education - educational company

EDIT: 

changed collection to collectibles
added more tags

**Reply by warengonzaga** [reply]
Suggest adding blockchain as tags like Polygon, Solana, Near, Tezos, Eth, etc...
This is because if they want to use our platform to look for NFT collections on the specific chain it would be helpful.

example: `nft, nft-collection, solana`

**Reply by warengonzaga** [reply]
> Suggest adding blockchain as tags like Polygon, Solana, Near, Tezos, Eth, etc... This is because if they want to use our platform to look for NFT collections on the specific chain it would be helpful.
> 
> example: `nft, nft-collection, solana`

@kquirapas here.

**Reply by jnebab** [reply]
> w2e = workout to earn p2e = play to earn m2e = move to earn pae = play and earn f2p = free to play nft = project that involves nft staking - project that provides staking metaverse - project that provides metaverse feature technology = project that provides technology innovation solution = a web 3 solution project service = a web 3 service project community = community engagement project dao = dao project guild = guild project defi = tackles decentralized finance wallet = web3 wallet collectibles= nft collection project gaming = gaming project arts = focus on arts streaming = streaming platform on web3 news = news content delivery multimedia = multimedia company marketplace = marketplace for nft launchpad = launchpad for projects organization = organization vc = venture capital company ai - ai oriented project education - educational company
> 
> EDIT:
> 
> changed collection to collectibles added more tags

We can start with these, I think. If in case, we have more to add. we can just do it every update

**Reply by kquirapas** [reply]
what if instead of predetermined tags we allow them input their own? @jnebab @itsnalla @warengonzaga

**Reply by chefallan** [reply]
> what if instead of predetermined tags we allow them input their own? @jnebab @itsnalla @warengonzaga

yeah prolly the best solution for the tags.

**Reply by jnebab** [reply]
my concern if we let them do self input tags is, it will make make the list of tags bloated due to mispellings and dashes. for example, some might put, nft-collection or  nftcollection or nft, collection. if we provide them predetermined tags, it will lessen the human error

**Reply by kquirapas** [reply]
Sige we can always add more later so let's go with predetermined then these are my final suggestions:

earn
nft
metaverse
dapp
service
community
dao
guild
defi
product
gaming
content (news site, blog site, etc)
organization
company
education

**Comment by kquirapas** [comment]
@jnebab @itsnalla @warengonzaga Self input tags nalang much better let them research on what the best tags is for their product :D

**Comment by kquirapas** [comment]
Sige we can always add more later so let's go with predetermined then these are my final suggestions:

earn
nft
metaverse
dapp
service
community
dao
guild
defi
product
gaming
content (news site, blog site, etc)
organization
company
education

@jnebab @warengonzaga @itsnalla

**Reply by chefallan** [reply]
LGTM

**Reply by warengonzaga** [reply]
question, why `earn`?

**Comment by kquirapas** [comment]
to earn, play to earn, etc.

so if its

gaming, then earn

they know its p2e 😆

**Reply by warengonzaga** [reply]
make sense to me at least.

**Reply by jnebab** [reply]
we will have tooltip text later on so I guess, making the tags shorter is much better, less space to consume and more to show

**Reply by warengonzaga** [reply]
let's go with that @jnebab

**Comment by kquirapas** [comment]
@jnebab @warengonzaga @itsnalla 

Final list of tags

earn
nft
metaverse
dapp
service
community
dao
guild
defi
product
gaming
content (news site, blog site, etc)
organization
company
education
location

**Reply by chefallan** [reply]
i'll update it and add the official tags on on our readme.

**Reply by warengonzaga** [reply]
all good to me
